CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_pendo v0.5.0
+## 🚨 Breaking Changes 🚨:
+- This is a breaking change due to changes made in the source package. (See [Source CHANGELOG](https://github.com/fivetran/dbt_pendo_source/blob/main/CHANGELOG.md)). 
```

Comment:
should we call out the end transform models affected by this change? i think the new field only comes up in `pendo__guide_event`